.github/workflows/platforms.yml

```diff
@@ -59,7 +59,10 @@ jobs:
             apt-get update
             apt-get install -y $(cat ./.github/packages/ubuntu-20.04-apt.txt)
             ./scripts/log-env.sh
-            meson build -Duse_fluidsynth=false --wrap-mode=nofallback
+            meson build \
+              -Duse_fluidsynth=false \
+              -Duse_mt32emu=false \
+              --wrap-mode=nofallback
```

Comment:
Just a related comment to MT32 on ARM - OpenSUSE provides mt32emu in Zypper, however it's build appears to be broken, as I was testing the `kc/mt32emu-floating-point-rendering-1` and was only getting garble on the Pi3.  
I was about to give up when I decided to try building mt32emu from source instead of using Zypper package.  Well, the source-built and optimized mt32emu worked perfectly :-). 
So this is just a comment that we will probably want to build mt32emu from source on these platforms as well.

Reply:
Glad to hear it worked :)

As for these lines you commented on here - I disabled mt32emu here because Docker images we use here don't have access to the external world and won't be able to download the sources - however I think we'll be able to fix this in the future. 

Meson keeps all downloaded files in `subprojects/packagecache` - we just need to populate this directory the same way we handle ccache and MSYS2 cache already - we won't be hammering WrapDB needlessly, and we'll be able to turn the features on on the wider range of CI jobs :)

Reply:
Thanks for that explanation!

> we just need to populate this directory the same way we handle ccache and MSYS2 cache already - we won't be hammering WrapDB needlessly, and we'll be able to enable the features on the wider range of CI jobs

Perfect!